<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Soren Harward &lt;stharward@gmail.com&gt;</author>
 <class>CheckpointEditWidget</class>
 <widget class="QWidget" name="CheckpointEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>113</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string>An editor for a CheckPoint constraint.  The user can set the track, album, or artist to be placed at a certain position in the playlist.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Checkpoint Constraint Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="TrackSelectWidget" name="trackSelector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Select the track that you want to use as the checkpoint.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Position">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Position:</string>
          </property>
          <property name="buddy">
           <cstring>timeEdit_Position</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTimeEdit" name="timeEdit_Position">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>The desired position of the track in the playlist, in hours, minutes, and seconds.  Note: because of Qt limitations, the maximum position that you can specify is 24 hours.</string>
            </property>
            <property name="displayFormat">
             <string>h:mm:ss</string>
            </property>
            <property name="time">
             <time>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Strictness">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>How strict the APG should be about placing the selected track at the specified time.</string>
          </property>
          <property name="text">
           <string>Match:</string>
          </property>
          <property name="buddy">
           <cstring>slider_Strictness</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_FuzzyPos">
            <property name="text">
             <string>fuzzy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Strictness">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>How strict the APG should be about placing the selected track at the specified time.</string>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ExactPos">
            <property name="text">
             <string>exact</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrackSelectWidget</class>
   <extends>QWidget</extends>
   <header>widgets/TrackSelectWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>timeEdit_Position</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
